addons/vehicles/initSettings.sqf

```diff
@@ -25,6 +25,6 @@
     "SLIDER",
     LSTRING(SpeedLimiterStep),
     ELSTRING(common,ACEKeybindCategoryVehicles),
-    [1, 10, 1, 0],
+    [1, 10, 5, 0],
     2
```

Comment:
Should we really use this? What if someone wants players to only be able to set speed limiter by 10ths or something. CBA wiki says it's not advisable to use as well.

Reply:
-1 might make sense, would keep all speeds to whole numbers